e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/validation/nature-of-business-employees-validation.spec.js

```diff
@@ -22,6 +21,8 @@ const {
 
 const baseUrl = Cypress.config('baseUrl');
 
+const expectedErrorsCorrectValidation = 2;
```

Comment:
This variable name is confusing, I don't think we should associate "correct" with "validation".

Reply:
reverted to just number